### PR TITLE
Add new ENV ROUNDCUBEMAIL_INSTALL_PLUGINS

### PR DESCRIPTION
**Pull request by williamdes**
Closes: https://github.com/roundcube/roundcubemail-docker/pull/104

This PR replaces #104 and uses another way of doing the changes that was documented on the README for years.

### CODE REVIEW

File: apache/docker-entrypoint.sh [hunk 1]
```diff
@@ -72,6 +72,25 @@ if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
   : "${ROUNDCUBEMAIL_TEMP_DIR:=/tmp/roundcube-temp}"
   : "${ROUNDCUBEMAIL_REQUEST_PATH:=/}"
 
+  if [ ! -z "${ROUNDCUBEMAIL_INSTALL_PLUGINS}" ]; then
+    echo "Installing plugins from the list"
+    echo "Plugins: ${ROUNDCUBEMAIL_INSTALL_PLUGINS}"
```
Comment (thomascube) [inline apache/docker-entrypoint.sh:77]: Wouldn't it be better to print `${ROUNDCUBEMAIL_PLUGINS}` here?

File: templates/docker-entrypoint.sh [hunk 2]
```diff
@@ -72,6 +72,25 @@ if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
   : "${ROUNDCUBEMAIL_TEMP_DIR:=/tmp/roundcube-temp}"
   : "${ROUNDCUBEMAIL_REQUEST_PATH:=/}"
 
+  if [ ! -z "${ROUNDCUBEMAIL_INSTALL_PLUGINS}" ]; then
+    echo "Installing plugins from the list"
+    echo "Plugins: ${ROUNDCUBEMAIL_INSTALL_PLUGINS}"
```
Comment (williamdes) [inline templates/docker-entrypoint.sh:77]:
```suggestion
    echo "Plugins: ${ROUNDCUBEMAIL_PLUGINS}"
```

File: fpm/docker-entrypoint.sh [hunk 3]
```diff
@@ -72,6 +72,25 @@ if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
   : "${ROUNDCUBEMAIL_TEMP_DIR:=/tmp/roundcube-temp}"
   : "${ROUNDCUBEMAIL_REQUEST_PATH:=/}"
 
+  if [ ! -z "${ROUNDCUBEMAIL_INSTALL_PLUGINS}" ]; then
+    echo "Installing plugins from the list"
+    echo "Plugins: ${ROUNDCUBEMAIL_INSTALL_PLUGINS}"
```
Comment (williamdes) [inline fpm/docker-entrypoint.sh:77]:
```suggestion
    echo "Plugins: ${ROUNDCUBEMAIL_PLUGINS}"
```

File: apache/docker-entrypoint.sh (hunk 1 above)
Comment (williamdes) [reply apache/docker-entrypoint.sh:77]: Indeed, I changed this

**Review by thomascube — APPROVED** [review]

**Comment by williamdes** [comment]
Updated by https://github.com/roundcube/roundcubemail-docker/pull/250

**Pull request merged**